src/main/java/mods/eln/node/NodeBase.kt

```diff
@@ -306,12 +306,12 @@ abstract class NodeBase {
         nbt.setByte("NBWrap", neighborWrapable)
     }
 
-    open fun multiMeterString(side: Direction): String {
-        return ""
+    open fun multiMeterString(side: Direction): String? {
+        return null
     }
 
-    open fun thermoMeterString(side: Direction): String {
-        return ""
+    open fun thermoMeterString(side: Direction): String? {
+        return null
```

Comment:
The real fix here is to go through and make sure we never pass null in the first place.